src/styles/index.scss

```diff
@@ -1,3 +1,136 @@
+$main-font: "Roboto", sans-serif;
+$main-color: #060b35;
+$second-color: #616070;
+$button-color: #00acdc;
```

Comment:
you can create a separate `variables.scss` file and store your colors there. (just import this file into `index.scss` using `@import`)